src/Paginator.php

```diff
@@ -791,18 +791,21 @@ public function normalizePageNumber($pageNumber)
     /**
      * Renders the paginator.
      *
-     * @param  \Laminas\View\Renderer\RendererInterface $view
+     * @param View\Renderer\RendererInterface|null $view
+     * @param array                                $params
      * @return string
      */
-    public function render(View\Renderer\RendererInterface $view = null)
-    {
+    public function render(
+        View\Renderer\RendererInterface $view = null,
+        array $params = []
```

Comment:
@froschdesign adding argument to public method which is not final/internal (also class is not final/internal) is a BC Break. Added label and milestone for that PR.

Reply:
Correct! 👍 
This pull request can wait and was only opened so that I do not forget the feature myself.

Reply:
Um.. adding an _optional_ parameter is not a BC break UNLESS it's done in an interface.

Yes, those extending the class _could_ add more arguments in their extensions. However, that then becomes undefined behavior when it comes to the original class being extended.

As such, this is not a BC break, and can be added in a new minor release.